src/v2/ion/i18nTransformer.js

```diff
@@ -270,26 +273,31 @@ const isWebAuthnAPIError = ( i18nKey ) => i18nKey.startsWith(WEBAUTHN_API_GENERI
  * @property {string[]} i18n.params
  */
 /**
- * - iff `message.i18n.key` exists and has value in 'login.properties', return the value.
- * - otherwise returns `message.message`
+ * - If `message.i18n.key` exists and has a value in 'login.properties'
+ *   through the given key or via I18N_OVERRIDE_MAPPINGS, return the value.
+ *
+ * - returns `message.message` otherwise
  *
  * @param {Message} message
  */
 const getMessage = (message) => {
   if (message.i18n?.key) {
-    const i18nKey = message.i18n.key;
+    const i18nKey = I18N_OVERRIDE_MAPPINGS[message.i18n?.key] ?? message.i18n.key;
```

Comment:
For instances where the API returns a key that we **do not want to use**, we can now override it to ensure v1 customizations stay intact.